mk/spksrc.common.mk

```diff
@@ -14,7 +14,7 @@ MSG = echo "===> "
 RUN = cd $(WORK_DIR)/$(PKG_DIR) && env $(ENV)
 
 # Pip command
-PIP ?= pip
+PIP ?= pip #define PIP in your makefile if you need pip3
 PIP_WHEEL = $(PIP) wheel --no-use-wheel --no-deps --wheel-dir $(STAGING_DIR)/wheelhouse
```

Comment:
spksrc is something to learn by example, getting to know the existing packages will teach you all the possibilities spksrc offers.
I agree documentation needs some love but until then, this is how to proceed :)